pkgs/applications/audio/vgmstream/default.nix

```diff
@@ -1,23 +1,29 @@
 { stdenv, lib, fetchFromGitHub, cmake, pkg-config
-, mpg123, ffmpeg, libvorbis, libao, jansson
+, mpg123, ffmpeg, libvorbis, libao, jansson, speex
 }:
 stdenv.mkDerivation rec {
-  pname   = "vgmstream";
-  version = "r1050-3448-g77cc431b";
+  pname = "vgmstream";
+  version = "r1702-5534-ab6139c11";
```

Comment:
Can we just drop the commit count and just use the release tag? That would make everything much easier.

Reply:
vgmstream doesn't really have release tags. Their page says:
> Please note that our versions are simply generated every now and then from the latest commits. Automated builds should be as stable as possible.

My plan was to simply use whatever version is current whenever `nixpkgs-update` runs.

Reply:
I don't like that. This means r-ryantm creates an update every run which would be every few days. Can we just rely on the tags in the repo or manual bumps?

Reply:
We could, but that goes against the mission "to make nixpkgs the most up-to-date repository of software in the world by the most ridiculous margin possible" ;)

On a more serious note, how do we handle other fast-moving packages that don't tag releases? Is there some sort of official policy on it?

Reply:
This just creates a lot of unnecessary commits which require a lot of reviewer resources with no real benefit. We do not need to be up to date to the day or hour.

> On a more serious note, how do we handle other fast-moving packages that don't tag releases? Is there some sort of official policy on it?

They are tagged unstable-XXXX-XX-XX and bumped at most once a week.

Reply:
Hmmm, I could modify the update script to only run if it's been a week (or some interval) since the last commit. Would that be acceptable?

I'd much prefer to keep the version format as-is (to match upstream) though. At the very least, even if the version is changed to unstable-XXXX-XX-XX, I'd like to keep the version string so it can be passed through as a `#define`.

Reply:
> Hmmm, I could modify the update script to only run if it's been a week (or some interval) since the last commit. Would that be acceptable?

Yeah, we could do that to prevent excessive updates.

Reply:
Finally got around to doing this. It now won't do anything if it's been less than a week between commits.